core/src/CMakeLists.txt

```diff
@@ -2,21 +2,57 @@ configure_file(CabanaCore_config.hpp.cmakein CabanaCore_config.hpp)
 
 #-----------------------------------------------------------------------------
 
-file(GLOB HEADERS_PUBLIC *.hpp)
-file(GLOB SOURCES *.cpp)
+set(HEADERS_PUBLIC
+  Cabana_AoSoA.hpp
+  Cabana_Core.hpp
+  Cabana_DeepCopy.hpp
+  Cabana_ExecutionPolicy.hpp
+  Cabana_LinkedCellList.hpp
+  Cabana_Macros.hpp
+  Cabana_MemberTypes.hpp
+  Cabana_NeighborList.hpp
+  Cabana_Parallel.hpp
+  Cabana_Slice.hpp
+  Cabana_SoA.hpp
+  Cabana_Sort.hpp
+  Cabana_Tuple.hpp
+  Cabana_Types.hpp
+  Cabana_VerletList.hpp
+  Cabana_Version.hpp
+  )
```

Comment:
Say I create a new header and forget to list it here.  Will it be caught when building and running the tests?

Reply:
It may although that is tough to tell because it is header only so it may just pull it in. It will likely show up more when external libraries link in and the missing header is not installed.

Reply:
No, it won’t, hence globbing and subfolders might be preferred!

Reply:
Lets push this as is with regards to the build system them and move this conversation to #55 - once we come to a conclusion there we can implement in a new PR